checkov/common/bridgecrew/integration_features/features/suppressions_integration.py

```diff
@@ -142,6 +145,14 @@ def _apply_suppressions_to_report(self, scan_report: Report) -> None:
         scan_report.failed_checks = still_failed_checks
         scan_report.passed_checks = still_passed_checks
 
+    @staticmethod
+    def _should_omit_check(applied_suppression: dict[str, Any]) -> bool:
+        if applied_suppression['isV1']:
```

Comment:
Why don't we check for the suppression type here?

Reply:
@mikeurbanski1 
because for v1 I currently don't omit (don't want to affect IAC / secrets/ SCA)